plugins/plugin-site/src/components/PluginHealthScore.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Progress } from 'antd';
```

Comment:
this seems like a large library to pull in for a <progress> element.

especially to keep with the bootstrap ui we have everywhere else

```
<div class="progress">
  <div class="progress-bar" role="progressbar" aria-valuenow={healthScore} aria-valuemin="0" aria-valuemax="100"></div>
</div>
```

Reply:
Here now I added reactstrap progress bar which is already in the dependencies